figanos/matplotlib/utils.py

```diff
@@ -307,7 +310,15 @@ def set_plot_attrs(
     if "cbar_units" in attr_dict:
         pass
 
-    return ax
+    if facetgrid:
+        if "suptitle" in attr_dict:
+            suptitle = get_attributes(attr_dict["suptitle"], xr_obj)
+            facetgrid.fig.suptitle(suptitle, y=1.05)
```

Comment:
It seems like placing the title above the plot with y>1 only works in notebooks:
https://stackoverflow.com/questions/55767312/how-to-position-suptitle#comment98213058_55768955
I have successfully used it in notebooks, but it fails for me in SciView/PyCharm.

Reply:
I can remove the y=1.05, but I have noticed that playing with the figsize also helps with suptitle